.github/workflows/ci.yml

```diff
@@ -111,6 +110,13 @@ jobs:
         with:
           submodules: true
           fetch-depth: 0
+      - name: Build GeoIP
+        run: |
+          git clone https://github.com/maxmind/geoip-api-c.git
```

Comment:
```suggestion
          git clone --depth 0 https://github.com/maxmind/geoip-api-c.git
```

Reply:
Hmm, [seems](https://github.com/owasp-modsecurity/ModSecurity/actions/runs/12613242971/job/35150863548?pr=3319#step:4:9) like `--depth 0` isn't a good value.

Even though I couldn't find any information in [official](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt-code--depthcodeemltdepthgtem) documentation, I found an [article](https://medium.com/@thriving_chiffon_gnu_714/exploring-git-clone-depth-828eb4bac6c4) when they said the minimum value is `1`.

What do you think?

Reply:
Yeah, I'm never sure :) Use `1` then.

Reply:
Done.